src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -732,7 +733,8 @@ private static UserAuthenticationModel GetUserFromToken(JwtSecurityToken jwtSecu
             {
                 IsAuthenticated = true,
                 ProviderClaims = new Dictionary<string,
-                List<string>>(), Iss = provider.IssuerKey,
+                List<string>>(),
```

Comment:
This line should have been on the lone above. It's part of the declaration of the Dictionary.